<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlpaintClass</class>
 <widget class="QMainWindow" name="AlpaintClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alpaint</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="titleBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>Alpaint</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="minimizeButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	border: 0px;
	color: white;
}

QToolButton:hover
{
	background-color: rgb(229, 229, 229);
	color: black;
}r</string>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="resizeButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	border: 0px;
	color: white;
}

QToolButton:hover
{
	background-color: rgb(229, 229, 229);
	color: black;
}r</string>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe MDL2 Assets</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	border: 0px;
	color: white;
}

QToolButton:hover
{
	background-color: rgb(232, 17, 35);
}r</string>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Canvas" name="canvas">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QFrame</extends>
   <header>canvas/Canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Alpaint.qrc"/>
 </resources>
 <connections/>
</ui>
